Assets/_BForBoss/_Character/Scripts/LensDistortionTool.cs

```diff
@@ -0,0 +1,35 @@
+using DG.Tweening;
+using UnityEngine.Rendering.Universal;
+
+namespace BForBoss
+{
+    /// <summary>
+    /// Tool to distort the lens.
+    /// WARNING/NOTE - Post Processing > Volume > LensDistortion - make sure intensity is turned on
+    /// For some reason there's no built in function to enable this automatically. 
```

Comment:
Note*
You MUST have LensDistortion > Intensity turned manually in inspector on for this to work.